### PR TITLE
GO-4232 Refactor InstallBundledObjects and createObjectType

### PR DESCRIPTION
**Pull request by KirillSto**
https://linear.app/anytype/issue/GO-4232/add-every-system-relations-to-recommended-relations-to-make-a-list-in

- introduce BundledObjectInstaller that incapsulates InstallBundledObjects method from creator
- fill recommendedRelations with predefined system relations
- exclude name relation from layout-specific

### CODE REVIEW

File: core/block/object/installer/bundle.go [hunk 1]
```diff
@@ -1,38 +1,78 @@
...
+
+func New() BundleObjectInstaller {
+	return &objectInstaller{}
+}
+
+func (i *objectInstaller) Init(a *app.App) error {
+	i.spaceService = app.MustComponent[space.Service](a)
+	i.store = app.MustComponent[objectstore.ObjectStore](a)
+	i.archiver = app.MustComponent[objectArchiver](a)
+	i.creator = app.MustComponent[objectcreator.Service](a)
```
Comment (deff7) [inline core/block/object/installer/bundle.go:57]: i.creatorService?

File: core/block/object/installer/bundle.go [hunk 2]
```diff
@@ -1,38 +1,78 @@
...
+	creator      objectcreator.Service
+}
+
+func New() BundleObjectInstaller {
+	return &objectInstaller{}
+}
+
+func (i *objectInstaller) Init(a *app.App) error {
+	i.spaceService = app.MustComponent[space.Service](a)
+	i.store = app.MustComponent[objectstore.ObjectStore](a)
```
Comment (deff7) [inline core/block/object/installer/bundle.go:55]: i.objectStore

File: core/block/object/installer/bundle.go (hunk 1 above)
Comment (KirillSto) [reply core/block/object/installer/bundle.go]: Renamed

File: core/block/object/installer/bundle.go (hunk 2 above)
Comment (KirillSto) [reply core/block/object/installer/bundle.go]: Renamed

**Review by deff7 — APPROVED** [review]

**Pull request merged**